docs/stock/technical-indicator-series.md

```diff
@@ -3,91 +3,37 @@ Technical indicators
-    
-*   [Slow Stochastic](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/slow-stochastic/)
-
-*   [Stochastic](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/stochastic/)
-    
-*   [Super Trend](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/supertrend/)
-    
-*   [Triple EMA (Exponential Moving Average)](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/tema/)
-    
-*   [TRIX](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/trix/)
-    
-*   [VbP (Volume by Price)](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/volume-by-price/)
-    
-*   [VWAP (Volume Weighted Average Price)](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/vwap/)
-    
-*   [Williams %R](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/williams-r/)
-    
-*   [WMA (Weighted Moving Average)](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/wma/)
-    
-*   [Zig Zag](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/zigzag/)
+Technical indicators require the [indicators/indicators.js](https://code.highcharts.com/stock/indicators/indicators.js) main module. The main module includes SMA (Simple Moving Average). Each technical indicator, except the SMA, is a separate module and should be loaded after the main module. 
+
+A full list of supported technical indicators could be divided into two main groups.  Overlays use the same scale and are plotted on the same xAxis and yAxis as the main series. The oscillator requires additional yAxis because they oscillate between different extremes.
+
+
+| Overlays  |Oscillators |
+| ------------- | ------------- |
+| [Acceleration Bands](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/acceleration-bands/) | [Absolute Price Oscillator](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/apo/) |
+| [Bollinger Bands](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/bollinger-bands/) | [A/D (Accumulation/Distribution)](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/accumulation-distribution/) |
+| [Double EMA (Exponential Moving Average)](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/dema/) | [Aroon](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/aroon/) |
```

Comment:
The official full name for DEMA is `Double Exponential Moving Average`. The same for TRIX - `Tripple Exp...`.
Please, correct the same in your other pull requests.

Reply:
I think you meant TEMA instead of TRIX?

Reply:
Yes, sorry my bad! TEMA is `Triple Exponential Moving Average`, TRIX is `Triple Exponential Average`. However, when doing small research now, I found different websites that conflict with each other. Some of them call TRIX `Triple Exponential Average`, but the rest call it `Triple Exponential Moving Average`. Please, make sure which one is correct (I assume the naming in my second sentence is correct).

Reply:
Our code says that `TRIX` stands for `Triple exponential average`, I think we should stick with that.
https://github.com/highcharts/highcharts/blob/cc1adf69558e1aea9b731c27e7c46d0a17911981/ts/Stock/Indicators/TRIX/TRIXIndicator.ts#L42